README.md

```diff
@@ -181,6 +181,43 @@ NAME                            READY   STATUS    RESTARTS   AGE
 policy-local-59854877c9-xwtfk   1/1     Running   0          2m38s
 ```
 
+### Podman Support
+
+Podman does not allow binding to privileged ports (e.g., 1-1024) unless it's
+running as root, which is not recommended.
+
+To connect to your service when using Podman, run `cloud-provider-kind` with the
+`-enable-lb-port-mapping` option. This configures the Envoy container with an
```

Comment:
```suggestion
`--enable-lb-port-mapping` option. This configures the Envoy container with an
```

Reply:
The `--help` option lists the options with a single dash. That's the only reason I used it that way. Happy to change though.

```
> cloud-provider-kind --help
Usage: cloud-provider-kind [options]

  -enable-lb-port-mapping
        enable port-mapping on the load balancer ports
  -enable-log-dumping
        store logs to a temporal directory or to the directory specified using the logs-dir flag
  -logs-dir string
        store logs to the specified directory
  -v int
        Verbosity level (default 2)
```